app/views/content/what-is-a-pgce.md

```diff
@@ -19,28 +26,64 @@ keywords:
   - State-school
 ---
 
-A postgraduate certificate in education (PGCE) and postgraduate diploma in education (PGDE) are academic qualifications you can get through teacher training.
+A postgraduate certificate in education (PGCE) is an academic qualification. You can get this through teacher training.
 
-## PGCE
+You do not need one to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.
 
-The course you take to get a PGCE is a mixture of learning education theory, and school placements where you will be put in a classroom to learn from experience.
+## The qualifications you need to teach in England
 
-Entry requirements for a PGCE course usually include an undergraduate degree or an equivalent qualification. The course will usually take 1 year to complete full time, or 2 years part time.
+You need qualified teacher status (QTS) to teach in most primary, secondary and special schools in England. You can get QTS through undergraduate or postgraduate teacher training.
 
-[Find a postgraduate teacher training course](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+[Find out more about qualified teacher status (QTS)](/what-is-qts).
 
-### Types of PGCE courses
+Many teacher training courses also offer you a postgraduate qualification such as a PGCE, but you do not need one to teach.
 
-You can do a PGCE course at a university, or through a school-led teacher training programme.
+## Teacher training courses
 
-You can do a PGCE in primary education, secondary education, or further/adult education.
+You can do a teacher training course that leads to:
 
-## PGDE
+* just QTS
+* just PGCE
+* QTS with PGCE
 
-A postgraduate diploma in education (PGDE) course is very similar to a PGCE, the difference being the amount of academic credits you are awarded at the end of your course.
+Getting just QTS means that you’re qualified to teach in most schools in England. It may also mean that your course fees are lower, you submit fewer assignments, and the course could be shorter.
 
-## Teaching qualifications you need
+Getting just a PGCE means you will not be qualified to teach in most primary, secondary and special schools in England.
 
-If you want to be a primary school teacher or secondary school teacher in a state school in England, you will need to get qualified teacher status (QTS). You don’t need a PGCE or PGDE to become a teacher, but many courses offer it alongside QTS.
+[Read about Nathan who chose to get QTS with PGCE](/blog/salaried-teacher-training-classroom-learning).
```

Comment:
Do we have a more recent blog post that would be relevant? Now that we have the banner at the top of the page re NQTs, the blog post feels noticeably old

Reply:
There is this [PGCE blog post[(https://getintoteaching.education.gov.uk/blog/what-to-expect-on-your-teacher-training) currently in the related content section. The only thing is, I don't think it's very clear whether it's talking about a PGCE only course, or a QTS with PGCE course.